test/autests/gold_tests/autest-site/proxy_http1.py

```diff
@@ -179,6 +179,22 @@ def wrap_socket(self, sock, *args, **kwargs):
     do_OPTIONS = do_GET
     do_options = do_GET
 
+    def relay_streaming(self, res):
```

Comment:
Where is this used?

Reply:
It's used when the length of the response isn't specified via a content-length. It's actually not currently used by any of our tests, but if we run into that situation, the interpreter will complain about the function not being defined. This is left over from the example proxy I based this off of. I'm pulling it in in case it is used. I've fixed the test that ran into this situation not to use that.